test/studies/diffusion/heat_2d_dist.skipif

```diff
@@ -0,0 +1 @@
+CHPL_COMM==none
```

Comment:
I think this file should be removed.  We typically don't skip tests that work in a configuration for the sake of saving time (which is the only reason I think this is probably here?).  If we wanted to do that, we'd do it through a `start_test --skip-multilocale-tests` run or the like.  Specifically, we don't want bugs to creep into codes in the `numLocales=1` configuration when they work there (which this does), and if someone was deprecating a feature that this test used, it'd be a shame to have their CHPL_COMM=none paratest skip over it.